src/main/java/net/william278/profanitycheckerapi/ProfanityChecker.java

```diff
@@ -91,4 +179,56 @@ public void dispose() throws IllegalStateException {
         close();
     }
 
+    private Set<String> getTextToCheck(String text) {
```

Comment:
Can we split the contents of this method up into a few different ones; one for each kind of filter we're applying to the text? Just to make it a bit manageable. A javadoc comment above each one, too, ideally.